SDK build fails on jitpack

## How to Reproduce
1. Try to get bunq Java SDK from Jitpack

## What Should Happen
1. The package is acquired smoothly

## What Happens
1. It fails, log show errors:
```
/home/jitpack/build/src/main/java/com/bunq/sdk/security/SecurityUtils.java:43: error: package org.jcp.xml.dsig.internal does not exist
import org.jcp.xml.dsig.internal.MacOutputStream;
                                ^
/home/jitpack/build/src/main/java/com/bunq/sdk/context/ApiContext.java:14: error: package com.sun.istack.internal does not exist
import com.sun.istack.internal.Nullable;
```

--- a/build.gradle
+++ b/build.gradle
@@ -2,6 +2,7 @@ group 'com.bunq.sdk'
 version '0.9.1'
 
 apply plugin: 'java'
+apply plugin: 'maven'
 sourceCompatibility = 1.8
 
 repositories {

--- a/src/main/java/com/bunq/sdk/context/ApiContext.java
+++ b/src/main/java/com/bunq/sdk/context/ApiContext.java
@@ -2,16 +2,15 @@ package com.bunq.sdk.context;
 
 import com.bunq.sdk.exception.BunqException;
 import com.bunq.sdk.json.BunqGsonBuilder;
-import com.bunq.sdk.model.generated.DeviceServer;
 import com.bunq.sdk.model.Installation;
 import com.bunq.sdk.model.SessionServer;
+import com.bunq.sdk.model.generated.DeviceServer;
 import com.bunq.sdk.model.generated.Session;
 import com.bunq.sdk.security.SecurityUtils;
 import com.google.gson.Gson;
 import com.google.gson.JsonObject;
 import com.google.gson.annotations.Expose;
 import com.google.gson.annotations.SerializedName;
-import com.sun.istack.internal.Nullable;
 import java.io.File;
 import java.io.IOException;
 import java.net.URI;
@@ -21,6 +20,7 @@ import java.util.Date;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import javax.annotation.Nullable;
 import org.apache.commons.io.FileUtils;
 
 /**

--- a/src/main/java/com/bunq/sdk/model/generated/Card.java
+++ b/src/main/java/com/bunq/sdk/model/generated/Card.java
@@ -160,7 +160,7 @@ public class Card extends BunqModel {
    */
   @Expose
   @SerializedName("pin_code_assignment")
-  private CardPinAssignment pinCodeAssignment;
+  private List<CardPinAssignment> pinCodeAssignment;
 
   public static BunqResponse<Card> update(ApiContext apiContext, Map<String, Object> requestMap,
       Integer userId, Integer cardId) {
@@ -386,11 +386,11 @@ public class Card extends BunqModel {
   /**
    * Array of Types, PINs, account IDs assigned to the card.
    */
-  public CardPinAssignment getPinCodeAssignment() {
+  public List<CardPinAssignment> getPinCodeAssignment() {
     return this.pinCodeAssignment;
   }
 
-  public void setPinCodeAssignment(CardPinAssignment pinCodeAssignment) {
+  public void setPinCodeAssignment(List<CardPinAssignment> pinCodeAssignment) {
     this.pinCodeAssignment = pinCodeAssignment;
   }
 

--- a/src/main/java/com/bunq/sdk/security/MacOutputStream.java
+++ b/src/main/java/com/bunq/sdk/security/MacOutputStream.java
@@ -1,0 +1,24 @@
+package com.bunq.sdk.security;
+
+import java.io.ByteArrayOutputStream;
+import javax.crypto.Mac;
+
+public class MacOutputStream extends ByteArrayOutputStream {
+
+  private final Mac mac;
+
+  MacOutputStream(Mac mac) {
+    this.mac = mac;
+  }
+
+  public void write(int oneByte) {
+    super.write(oneByte);
+    this.mac.update((byte) oneByte);
+  }
+
+  public void write(byte[] bytes, int offset, int length) {
+    super.write(bytes, offset, length);
+    this.mac.update(bytes, offset, length);
+  }
+
+}

--- a/src/main/java/com/bunq/sdk/security/SecurityUtils.java
+++ b/src/main/java/com/bunq/sdk/security/SecurityUtils.java
@@ -40,7 +40,6 @@ import org.apache.http.client.methods.HttpPost;
 import org.apache.http.client.methods.HttpPut;
 import org.apache.http.client.methods.HttpUriRequest;
 import org.apache.http.util.EntityUtils;
-import org.jcp.xml.dsig.internal.MacOutputStream;
 
 /**
  * Static lib containing methods for handling encryption.
@@ -333,7 +332,8 @@ public final class SecurityUtils {
       Mac mac = Mac.getInstance(ALGORITHM_MAC);
       mac.init(key);
       mac.update(initializationVector);
-      BufferedSink bufferedSink = Okio.buffer(Okio.sink(new MacOutputStream(mac)));
+      MacOutputStream outputStream = new MacOutputStream(mac);
+      BufferedSink bufferedSink = Okio.buffer(Okio.sink(outputStream));
       bufferedSink.write(requestBytes);
       bufferedSink.emit();
       bufferedSink.flush();

